### DISCUSSION TITLE
Teletype for Atom - GDPR Compliant?

### DISCUSSION THREAD

**Discussion by asgersong** [community/community#36796] [Other Feature Feedback, Questions, & Ideas]
I have not been able to find a separate page that describes Github's text-editor Atom's compliancy with GDPR and by extension Teletype. Which brings me here.. Is Teletype for Atom GDPR-compliant? If so, I would love to see the page that verifies this.

**Comment by siddhpant** [comment]
Atom has shut down.

https://github.blog/2022-06-08-sunsetting-atom/

**Comment by ossanna16** [comment]
Thank you for asking a good question @asgersong, I'm glad you found our community 🙂
